cumulus/ansible/tasks/cluster.py

```diff
@@ -12,24 +16,25 @@ def get_playbook_path(name):
                         "playbooks/" + name + ".yml")
 
 
-def run_playbook(playbook, inventory, extra_vars=None):
-    extra_vars = {} if extra_vars is None else extra_vars
+def run_playbook(playbook, inventory, extra_vars=None,
+                 verbose=None, env=None):
+
+    env = env if env is not None else os.environ.copy()
+
+    cmd = ["ansible-playbook", "-i", inventory]
+
+    if verbose is not None:
+        cmd.append("-%s" % ("v" * verbose))
 
-    stats = callbacks.AggregateStats()
+    if extra_vars is not None:
+        cmd.extend(["--extra-vars", json.dumps(extra_vars)])
 
-    pb = ansible.playbook.PlayBook(
-        playbook=playbook,
-        inventory=inventory,
-        callbacks=callbacks.PlaybookCallbacks(verbose=1),
-        runner_callbacks=callbacks.PlaybookRunnerCallbacks(stats, verbose=1),
-        stats=stats,
-        extra_vars=extra_vars
-    )
-    # Note:  can refer to callback.playbook.extra_vars  to get access
-    # to girder_token after this point
-    results = pb.run()
+    cmd.append(playbook)
 
-    return results
+    p = subprocess.Popen(cmd, env=env, stdout=subprocess.PIPE)
+
```

Comment:
@cjh1 we can grab stderr,  but this is just for output to the celery log. First,  is it nessisary? If so - If we pull it,  should we log it before or after stdout?